src/components/app-layout/constants.js

```diff
@@ -1,2 +1,4 @@
 export const sidebarWidthXl = 400
 export const sidebarWidth = 340
+export const navigationWidthXl = 280
```

Comment:
Usually, we take constants from [grape-theme](https://github.com/ubergrape/grape-theme/blob/master/src/sizes.js). But as we already have `sidebarWidthXl` and `sidebarWidth` there it's ok to two more new. @nikgraf just letting you know where we have constants.

Reply:
makes sense! thanks for the insights